library/src/main/kotlin/it/krzeminski/githubactions/actions/CustomAction.kt

```diff
@@ -0,0 +1,24 @@
+package it.krzeminski.githubactions.actions
+
+/**
+ * CustomAction can be used when there is no type-safe wrapper action
+ * and a quickly untyped wrapper is needed to fill the blank.
+ *
+ * Consider contributing a proper wrapper to the library, see [WrappersToGenerate.kt]
```

Comment:
This KDoc reference doesn't work, let's just skip it
```suggestion
 * Consider contributing a proper wrapper to the library, see WrappersToGenerate.kt in wrapper-generator module.
```

Reply:
Done